.github/CODEOWNERS

```diff
@@ -63,37 +81,37 @@ platforms/core-configuration/kotlin-dsl-tooling-builders/   @gradle/bt-tide
+platforms/core-configuration/declarative-dsl-internal-utils/        @gradle/bt-declarative-gradle-project
 
 # Core automation platform (core/runtime)
-platforms/core-runtime/                                     @gradle/bt-core-runtime-maintainers
-platforms/core-runtime/build-operations/                    @gradle/bt-core-runtime-maintainers @gradle/bt-execution @gradle/dv-integrations-team
-platforms/core-runtime/functional/                          @gradle/bt-core-runtime-maintainers @gradle/bt-execution @bamboo
-platforms/core-runtime/files/                               @gradle/bt-core-runtime-maintainers @gradle/bt-execution @gradle/dv-integrations-team
-platforms/core-runtime/service-*/                           @gradle/bt-core-runtime-maintainers @alllex
+platforms/core-runtime/                                     @gradle/bt-cortex
+platforms/core-runtime/build-operations/                    @gradle/bt-cortex @gradle/dv-integrations-team
+platforms/core-runtime/files/                               @gradle/bt-cortex @gradle/dv-integrations-team
 
 # Core automation platform (core/execution)
-platforms/core-execution/                                   @gradle/bt-execution
-platforms/core-execution/build-cache/                       @gradle/bt-execution @gradle/dv-integrations-team
-platforms/core-execution/build-cache-base/                  @gradle/bt-execution @gradle/dv-integrations-team
-platforms/core-execution/build-cache-http/                  @gradle/bt-execution @gradle/dv-integrations-team
-platforms/core-execution/build-cache-packaging/             @gradle/bt-execution @gradle/dv-integrations-team
-platforms/core-execution/build-cache-spi/                   @gradle/bt-execution @gradle/dv-integrations-team
-platforms/core-execution/hashing/                           @gradle/bt-execution @gradle/dv-integrations-team
-platforms/core-execution/snapshots/                         @gradle/bt-execution @gradle/dv-integrations-team
+platforms/core-execution/                                   @gradle/bt-cortex
+platforms/core-execution/build-cache/                       @gradle/bt-cortex @gradle/dv-integrations-team
+platforms/core-execution/build-cache-base/                  @gradle/bt-cortex @gradle/dv-integrations-team
+platforms/core-execution/build-cache-http/                  @gradle/bt-cortex @gradle/dv-integrations-team
+platforms/core-execution/build-cache-packaging/             @gradle/bt-cortex @gradle/dv-integrations-team
+platforms/core-execution/build-cache-spi/                   @gradle/bt-cortex @gradle/dv-integrations-team
+platforms/core-execution/hashing/                           @gradle/bt-cortex @gradle/dv-integrations-team
+platforms/core-execution/snapshots/                         @gradle/bt-cortex @gradle/dv-integrations-team
+
+# Other core modules
+subprojects/core                                            @gradle/bt-cortex
+subprojects/core-api                                        @gradle/bt-cortex
```

Comment:
❓ It is definitely a great incentive to untangle these, but I'm afraid `bt-cortex` team will be overwhelmed with code reviews in the area in the meantime. Can we keep these unowned for now?

Reply:
➕ Good point, @mlopatkin.